.eslintrc.js

```diff
@@ -1,29 +1,37 @@
 module.exports = {
     root: true,
     parser: '@typescript-eslint/parser',
-    plugins: [
-        '@typescript-eslint',
-    ],
+    plugins: ['@typescript-eslint'],
```

Comment:
I think we also need to add `'prettier'` to the plugins array here, and `'prettier/prettier': 'error'` to the rules object below.

From https://github.com/prettier/eslint-plugin-prettier:
```
{
  "plugins": ["prettier"],
  "rules": {
    "prettier/prettier": "error"
  }
}
```

Reply:
I think we may also want to add https://github.com/yannickcr/eslint-plugin-react. If we do, we'll have to add it and the prettier plugin line for compatibility with it.
In `plugins`: `'react'`
In `extends`: `plugin:react/recommended` and `prettier/react`

Reply:
@mturley, actually https://github.com/prettier/eslint-plugin-prettier#recommended-configuration says that adding `plugin:prettier/recommended` automatically sets `prettier/prettier:error` in rules.

Also the react plugin is going to wait once we have prettier in place because that's will require to fix lots of files.

Reply:
Ah got it, I didn't see that! Perfect, thanks.

Sounds good on the React plugin. We should follow up on that, there are some really useful rules in there.